.coveragerc

```diff
@@ -91,6 +91,8 @@ omit =
     homeassistant/components/bbox/sensor.py
     homeassistant/components/bh1750/sensor.py
     homeassistant/components/bitcoin/sensor.py
+    homeassistant/components/bitvavo/__init__.py
```

Comment:
You have a `test_init.py` so why omit it?

Reply:
Good find, will fix, as the other ones... thanks